deploy/determined_deploy/aws/templates/govcloud.yaml

```diff
@@ -0,0 +1,708 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+Description: Determined Template
+Mappings:
+  RegionMap:
+    us-gov-east-1:
+      Master: ami-af9379de 
+      Agent: ami-0b71e56244b8ac1cb
+    us-gov-west-1:
+      Master: ami-a6e8d2c7 
```

Comment:
Until we have a good answer to this (https://bugs.launchpad.net/cloud-images/+bug/1917080), could you add a blurb to the tools/scripts/README.md about what to manually look for and update when we run refresh-ubuntu-amis.py?

Reply:
Yup just added a blurb there.